### PR TITLE
Adding YOURLS' "user password encryption" support

### PR DESCRIPTION
**Pull request by pchiu33**

### CODE REVIEW

**Comment by LeoColomb** [comment]
Hey @pchiu33, thanks for your PR! Very appreciated! 👍 
What is your specific motivation with this change? Thx! 😃

**Comment by pchiu33** [comment]
Hi @LeoColomb, happy to contribute!
This change is because the YOURLS feature that encrypts user passwords is broken (there is a message window saying so after login in the admin UI).

**Comment by pchiu33** [comment]
I also added some fix in Dockerfile that would prevent the container to properly start.

**Comment by pchiu33** [comment]
Ultimately, these contributions - and later ones are because I'm considering running YOURLS for some projects.

**Comment by LeoColomb** [comment]
@pchiu33 Thanks a lot, let me review this! 😃 🍭

File: docker-entrypoint.sh [hunk 1]
```diff
@@ -81,6 +81,9 @@ if [ "$haveConfig" ]; then
     if [ ! -e /var/www/html/user/config.php ]; then
         cp /var/www/html/config-docker.php /var/www/html/user/config.php
         chown www-data:www-data /var/www/html/user/config.php
+        if [ "${YOURLS_USER}" -a "${YOURLS_PASS}" ]; then
+          sed -i "s/  getenv('YOURLS_USER') ? getenv('YOURLS_USER') : 'root' => getenv('YOURLS_PASS') ? getenv('YOURLS_PASS') : 'blah',/  '${YOURLS_USER}' => '${YOURLS_PASS}',/g" /var/www/html/user/config.php
+        fi
```
Comment (LeoColomb) [inline docker-entrypoint.sh]: Yes, thanks, SGTM!

File: Dockerfile.template [hunk 2]
```diff
@@ -12,7 +12,7 @@ RUN set -ex; \
     ; \
     rm -rf /var/lib/apt/lists/*; \
     \
-    docker-php-ext-install mbstring opcache pdo pdo_mysql
+    docker-php-ext-install mbstring opcache pdo pdo_mysql mysqli
```
Comment (LeoColomb) [inline Dockerfile.template]: Hmm, I would prefer to keep pdo only, but OK, may be easier to start from scratch with mysqli.

**Comment by pchiu33** [comment]
update.sh done

**Pull request merged**